src/twisted/python/_pydoctor.py

```diff
@@ -114,9 +115,13 @@ def _getURLAsHEAD(self, url):
         @type url: L{str}
 
         @return: The response for the HEAD method.
-        @rtype: urllib2 response
+        @rtype: urllib2 response or L{None}
         """
-        return urllib2.urlopen(HeadRequest(url))
+        try:
+            return urllib2.urlopen(HeadRequest(url))
+        except Exception as e:
+            print("Error opening {}: {}".format(url, e))
```

Comment:
I'll open a separate ticket for that later today since the current ptoblem is specific to Activestate.
In order to report to them I need to create an account on their site to log into their forums.